src/filesystem/DirectoryCache.cpp

```diff
@@ -63,10 +62,10 @@ bool CDirectoryCache::GetDirectory(const std::string& path, std::vector<kodi::vf
     const ItemListRecord& record = itItemList->second;
 
     // Check timestamp for stale data
-    const int64_t timestamp = record.first;
-    const int64_t expires = timestamp + DIRECTORY_LIFETIME_MS;
+    const std::chrono::steady_clock::time_point timestamp = record.first;
+    const std::chrono::steady_clock::time_point expires = timestamp + DIRECTORY_LIFETIME;
 
-    if (expires <= P8PLATFORM::GetTimeMs())
+    if (expires <= std::chrono::steady_clock::now())
```

Comment:
@AlwinEsch IIRC, you recently introduce a binary addon API function "GetTimeMs". Why do we need this? As you can see here,  "std::chrono::steady_clock::now()" does the same and no special Kodi API needed...

Reply:
Has overseen the new C++11, there are so much new parts inside that I still not know all.

Reply:
Could you please revert that API change? I think we should not bloat our API with stuff that can be realized with the c++ standard library.

Reply:
@AlwinEsch I opened https://github.com/xbmc/xbmc/pull/16035 to remove that addon API function.